cppwinrt/code_writers.h

```diff
@@ -158,8 +158,15 @@ namespace cppwinrt
         }
     }
 
+    template<std::size_t indents = 0>
```

Comment:
Rather than nesting namespaces, just use `namespace std::ranges` in a distinct top-level code gen block. That would be consistent with how the existing namespace are generated.